sonar-project.properties

```diff
@@ -26,6 +26,7 @@ sonar.coverage.exclusions=metrics_utility/test/**,metrics_utility/debug_utils.py
 # Manages code duplication sensitivity.  Default is 100.  Higher number means less sensitive.
 sonar.cpd.python.minimumTokens=150
 sonar.cpd.python.minimumLines=300
+sonar.cpd.exclusions=metrics_utility/test/**
```

Comment:
There's already a `sonar.cpd.exclusions` on line 43, I guess you'll want to merge the values together?